### PR TITLE
update README.md

### LINKED ISSUE #209: Update README.md instructions
**Is your feature request related to a problem? Please describe.**
Current instructions on the README.md is a bit confusing

**Describe the solution you'd like**
for example, using "Replace the ${client-id} and ${client-secret} with the values from the above linkedin social app"
instead of "Replace the client-id and client-secret with the values from the above linkedin social app" would be helpful.
also, we can provide a sample code.

for an example:
Screenshot 2021-11-15 at 22 18 13
(all username, passwords etc above are dummy values)

and also I suggest providing [SSH](https://ourtechroom.com/tech/https-vs-ssh-in-git/) method for git cloning. 
Using `git clone git@github.com:<your profile name>/scholarx.git` instead of `git clone https://github.com/<your profile name>/scholarx`

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose
The purpose of this PR is to fix #209  

## Goals
Avoid confusion with the instructions given in the README file

## Approach
added examples with dummy values

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
JDK v17.0.1
MacOS 12.0.1

### CODE REVIEW

**Comment by Madhawa97** [comment]
I didn't update the git clone URL to an SSH. is it necessary?

File: README.md [hunk 1]
```diff
@@ -2,40 +2,63 @@
...
-**Prerequisites**
+###Prerequisites
 * Java 
 * Maven
 * MySQL
 * Linkedin Social Login App
 * Gmail Account with an App Password 
 
-**Setup Linkedin Social Login App**
+###Setup Linkedin Social Login App
```
Comment (anjula-sack) [inline README.md:14]: `###Setup Linkedin Social Login App` I think there needs to be space-like `### Setup Linkedin Social Login App`, can you check again?

File: README.md [hunk 2]
```diff
@@ -2,40 +2,63 @@
...
 
-**Setup a gmail account with an app password**
+###Setup a gmail account with an app password
 1. Create a new gmail account if you don't have one already
 2. Enable Two Factor Authorisation 
-3. Generate a new `App Password` ([help?](https://support.google.com/mail/answer/185833?hl=en-GB)) 
+3. Generate a new `App Password` ([help?](https://support.google.com/mail/answer/185833?hl=en-GB))  
 
-**Steps**
+###Run Locally
```
Comment (anjula-sack) [inline README.md:25]: same here

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by anjula-sack** [comment]
> I didn't update the git clone URL to an SSH. is it necessary?

no

File: README.md (hunk 1 above)
Comment (Madhawa97) [reply README.md:14]: Sure. My bad

**Comment by Madhawa97** [comment]
did the requested changes

**Review by Piumal1999 — APPROVED** [review]
Looks good. Thank you @Madhawa97

**Review by anjula-sack — APPROVED** [review]
Thank you! @Madhawa97

**Pull request merged**